.github/workflows/maven.yml

```
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        java: [8, 11, 14, 15, 16, 17-ea]
+        java: [8, 11]
         jdk: [adopt, zulu]
```

adopt and zulu are both hotspot, no major differences.
adopt is ambiguous anyway, because you actually mean "adopt-hotspot".
Therefore I would recommend:

```suggestion
        jdk: ['adopt-openj9', zulu]
```